api/v1alpha1/oadp_types.go

```diff
@@ -97,7 +97,7 @@ type VeleroConfig struct {
 	DefaultVolumesToFSBackup *bool `json:"defaultVolumesToFSBackup,omitempty"`
 	// Specify whether CSI snapshot data should be moved to backup storage by default
 	// +optional
-	SnapshotMoveData *bool `json:"snapshotMoveData,omitempty"`
+	DefaultSnapshotMoveData *bool `json:"defaultSnapshotMoveData,omitempty"`
```

Comment:
This change can break updates

Would suggest adding this to this list https://github.com/openshift/oadp-operator/issues/1311

I believe the harmless way to do this, would be create the new field (`DefaultSnapshotMoveData`) and deprecate the older one (`SnapshotMoveData`)

Reply:
or we can just change the server flag I suppose so it's correct.  
that's a one liner.

Reply:
You've become the anti crd change man after encountering the enums once 😅.

If this field was guaranteed to crash velero pod, which it was, it meant no one was using it in production. This I believe this field is safe to change. We can also just keep crd and only make minimal code change

Reply:
hmm I think this case you can change the field name, because this was recently added https://github.com/openshift/oadp-operator/pull/1280/files (so, we did not release a DPA with this field yet, so, we can its name)